f8a_report/v2/report_generator.py

```diff
@@ -156,33 +132,21 @@ def analyse_stack(self, stacks_data, report_template) -> dict:
             try:
                 if len(analysed_dependencies) == 0:
                     continue
-                stack_info_template['ecosystem'] = ecosystem
                 self.total_stack_requests['all'] += 1
                 self.total_stack_requests[ecosystem] += 1
 
-                stack_info_template['stack'] = self.normalize_deps_list(
+                normalised_stacks = self.normalize_deps_list(
                     analysed_dependencies)
 
-                self.all_deps[ecosystem].append(stack_info_template['stack'])
-                stack_str = ','.join(stack_info_template['stack'])
+                self.all_deps[ecosystem].append(normalised_stacks)
+                stack_str = ','.join(normalised_stacks)
```

Comment:
why is normalised_stacks present in 2 places?

Reply:
`self.all_deps[ecosystem].append(normalised_stacks)`
is basically used in **summary section** to calculate  `top_dep` 
Here: https://github.com/fabric8-analytics/f8a-stacks-report/blob/5a43f43b9ce8094a56cf5c231f585fcc6e2680a4/f8a_report/helpers/report_helper.py#L413-L414 

and `unique_dependencies_with_frequency` 
https://github.com/fabric8-analytics/f8a-stacks-report/blob/5a43f43b9ce8094a56cf5c231f585fcc6e2680a4/f8a_report/helpers/report_helper.py#L404-L405 

Similarly 
`stack_str = ','.join(normalised_stacks)`

is being later used to calculate `unique_stacks_with_recurrence_count` for Report Summary Buildup